tests/Jellyfin.Naming.Tests/EpisodePathParserTest.cs

```diff
@@ -31,24 +32,24 @@ public void ParseEpisodesCorrectly(string path, string name, int season, int epi
 
         [Theory]
         [InlineData("/media/Foo/Foo 889", "Foo", 889)]
-        [InlineData("/media/Foo/[Bar] Foo Baz - 11 [1080p]", "Foo Baz", 11)]
+        [InlineData("/media/Foo/[Bar] Foo Baz - 11 [1080p]", "Foo Baz", 11)]
```

Comment:
That would explain the issues. What character was used before?

Reply:
No idea, haven’t checked yet.

Reply:
It was a no break space, whatever that means.